examples/Db2/README.md

```diff
@@ -0,0 +1,87 @@
+
+# Terraform Module to install and configure DB2 on Openshift
+
+**Module Source**: `github.com/ibm-hcbt/terraform-ibm-cloud-pak.git//modules/Db2`
+
+## Set up access to IBM Cloud
+
+If running these modules from your local terminal, you need to set the credentials to access IBM Cloud.
+
+Go [here](../../CREDENTIALS.md) for details.
+
+### Download required license files
+
+Download required license files from [IBM Internal Software Download](https://w3-03.ibm.com/software/xl/download/ticket.wss) or [IBM Passport Advantage](https://www.ibm.com/software/passportadvantage/) into the  `./files` folder
+```bash
+DB2:
+Part Number : CNB21ML
+Filename    : DB2_AWSE_Restricted_Activation_11.5.zip
+```
+Note: the license key is required only for Advanced DB2 installation
+
+
+
+## Provisioning this module in a Terraform Script
+
+See the example [here](../../examples/Db2) on how to provision this module.
+
+## Input Variables
+
+| Name                       | Description                                                            | Default                | Required |
+| ---------------------------|------------------------------------------------------------------------|------------------------|----------|
+| `ibmcloud_api_key`         | IBM Cloud API key: https://cloud.ibm.com/docs/account?topic=account-userapikey#create_user_key                                                    |                        | Yes      |
+| `resource_group`           | Region where the cluster is created. Managing resource groups: https://cloud.ibm.com/docs/account?topic=account-rgs&interface=ui        | `default`              | Yes      |
```

Comment:
The default in the `variables.tf` file is `cloud-pak-sandbox`

Reply:
Updated to `cloud-pak-sandbox`